source/renderer/app/api/addresses/types.js

```diff
@@ -1,22 +1,29 @@
 // @flow
 import WalletAddress from '../../domains/WalletAddress';
 
-export type Address = {
+export type AddressState = 'used' | 'unused';
+
+export type GetAddressesRequestQueryParams = {
+  state: AddressState,
+};
+
+export type AdaAddress = {
   id: string,
-  state: 'used' | 'unused',
+  state: AddressState,
 };
 
-export type Addresses = Array<Address>;
+export type AdaAddresses = Array<AdaAddress>;
+
+export type GetAddressesRequest = {
```

Comment:
In order to keep consistency, i believe it will be ok to rename this type to GetAdaAddressesRequest, right?

Reply:
I agree with keeping it consistent, but I'm still unclear as to why the `Ada` prefix is necessary instead of just calling it `Address` and `Addresses`. We only have currency in the wallet and that's `ADA` so of course it's an ADA compatible address. The only real differentiator I foresee bringing in a new differentiator are stake pool addresses. These aren't the same type of address as the standard wallet addresses we've been handling so far (I may be wrong though, but even so stake pool addresses are still used with ADA 😆). Not trying to make a big deal about this, but I'm dwelling on it with the intention of standardizing the naming conventions we use. I've been refactoring the tests directories into domains, I refactored the api into domains back when we were working on the V1 integration, and now I'm doing the same for storybook. I don't mind changing the `Addresses` domain name and types, but considering it is meant to be consistent across all sections of the wallet, we need to think carefully about these names and whether or not the V2 API is going to require differentiations in the current set of domain names we're using.

Reply:
@MarcusHurney regarding to this and previous comments, I will revert AdaAddress types back to just `Address`